modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,152 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The `UserDefinedNetwork` and `NetworkAttachmentDefinition` custom resources (CRs) provide cluster administrators and users the ability to create customizable network configurations and define their own network topologies, ensure network isolation, manage IP addressing for workloads, and leverage advanced network features. A third CR, `ClusterUserDefinedNetwork`, is also available, which allows administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level. Both user-defined networks and network attachment definitions can serve as both the primary and secondary network interface, and each support `layer2` and `layer3` topologies; a third network topology, Localnet, is also supported with network attachment definitions with secondary networks. 
+
+[NOTE]
+====
+As of {product-title} 4.18, the Localnet topology is unavailable for use with the `UserDefinedNetwork` leveraging primary networks and `ClusterUserDefinedNetwork` APIs. It is available for use with network attachment definitions that leverage secondary networks only.
+====
+
+The following section highlights the supported features of the `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs when they are used as either the primary or secondary network. A separate table for the `ClusterUserDefinedNetwork` CR is also included. 
+ 
+.Primary network support matrix
```

Comment:
For these three tables:

Something is wrong with the format - in the preview the left cell/column has not bottom line. 

I would posit that the left cell/column is actually adding unnecessary complexity here. If that cell applies to the whole table, work int into the table tile and get rid of the cell spanning all the rows. This should be easier to maintain and take less horizontal space on the page, making it easier to read. E.g. 
![image](https://github.com/user-attachments/assets/10532aef-fa58-4c48-9b9c-e60b337d0e62)

Reply:
Good call. I think the original thought was that UDN and NAD would have different features (and they do, to a degree) but this makes more sense.